### PR TITLE
28 add sitspot

### PR DESCRIPTION
**Pull request by Amoodaa**
closing bc of git merge issues #69 
relates #28, uploading using cloudinary, multer, FormData

### CODE REVIEW

File: client/src/pages/addNewSitSpot/index.css [hunk 1]
```diff
@@ -0,0 +1,110 @@
+.add-place__wrapper {
+  padding: 0 1.5rem;
+}
+
+.add-place__header {
+  background-image: url('../../assets/images/home-background.jpg');
```
Comment (SajedaIsmail) [inline client/src/pages/addNewSitSpot/index.css]: try to use another image for the background to let the header be clear

File: client/src/pages/addNewSitSpot/index.js [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+  };
+
+  handleChange = (value, dataKey) => {
+    this.setState(prevState => ({
+      ...prevState,
+      data: { ...prevState.data, [dataKey]: value },
+    }));
+  };
+
+  next = () =>
```
Comment (SajedaIsmail) [inline client/src/pages/addNewSitSpot/index.js]: Dont let the user go to the next before doing the current step!

File: server/app.js [hunk 3]
```diff
@@ -1,3 +1,7 @@
+if (process.env.NODE_ENV === 'development' || process.env.NODE_ENV === 'test')
+  // eslint-disable-next-line global-require, import/no-extraneous-dependencies
+  require('env2')('.env');
+
 const { join } = require('path');
 const express = require('express');
```
Comment (SajedaIsmail) [inline server/app.js]: delete this line

File: client/src/pages/addNewSitSpot/index.css (hunk 1 above)
Comment (Amoodaa) [reply client/src/pages/addNewSitSpot/index.css]: images can be changed later, all links in the project will be redone next week

File: client/src/pages/addNewSitSpot/index.js (hunk 2 above)
Comment (Amoodaa) [reply client/src/pages/addNewSitSpot/index.js]: good point but we ill leave it to asem.. im scared to touch his component code

File: server/app.js (hunk 3 above)
Comment (Amoodaa) [reply server/app.js]: why?

**Pull request merged**